### PR TITLE
Add zkSynk, remove rinkeby from documentation

### PR DESCRIPTION
**Pull request by BjMrq**
Add zkSynk, remove rinkeby from documentation

### CODE REVIEW

File: packages/sentinel/README.md [hunk 1]
```diff
@@ -166,13 +166,13 @@ There are two types of sentinels, `BLOCK` and `FORTA`. For more information on w
 
 To create a new sentinel, you need to provide the type, network, name, pause-state, conditions, alert threshold and notification configuration. This request is exported as type `CreateSentinelRequest`.
 
-An example for a `BLOCK` sentinel is provided below. This sentinel will be named `MyNewSentinel` and will be monitoring the `renounceOwnership` function on the `0x0f06aB75c7DD497981b75CD82F6566e3a5CAd8f2` contract on the Rinkeby network.
+An example for a `BLOCK` sentinel is provided below. This sentinel will be named `MyNewSentinel` and will be monitoring the `renounceOwnership` function on the `0x0f06aB75c7DD497981b75CD82F6566e3a5CAd8f2` contract on the goerli network.
```
Comment (shahnami) [inline packages/sentinel/README.md:169]: Should probably change `0x0f06aB75c7DD497981b75CD82F6566e3a5CAd8f2` to an existing Goerli contract

File: examples/pause-proposal/index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ require('dotenv').config();
 const { AdminClient } = require('defender-admin-client');
 
 const address = '0xB07b1C80371915dEFd254d1C57BeF2bDe6D3b610';
```
Comment (shahnami) [inline examples/pause-proposal/index.js:5]: And here

**Review by shahnami — APPROVED** [review]
Looks good, but ideally you should change the example addresses as well.

**Comment by BjMrq** [comment]
Good point! Deployed a dummy greeter contract ownable not verified [0x2d92Ee41273900b1721a0c5e3d3f53e973b92E79](https://goerli.etherscan.io/address/0x2d92Ee41273900b1721a0c5e3d3f53e973b92E79#code) should cover use case for "will be monitoring the `renounceOwnership`"

**Comment by BjMrq** [comment]
Turns out replacing each address with appropriate contract is more work than expected, will cancel this MR

**Pull request closed without merging**